project/ProjectPlugin.scala

```diff
@@ -84,20 +136,21 @@ object ProjectPlugin extends AutoPlugin {
         organizationEmail = "hello@47deg.com"
       ),
       orgLicenseSetting := ApacheLicense,
-      scalaVersion := "2.11.11",
+      scalaVersion := "2.12.10",
       scalaOrganization := "org.scala-lang",
       javacOptions ++= Seq("-encoding", "UTF-8", "-Xlint:-options"),
       fork in Test := false,
       parallelExecution in Test := false,
       cancelable in Global := true,
-      headers := Map(
-        "scala" -> (HeaderPattern.cStyleBlockComment,
-        s"""|/*
+      headerLicense := Some(
+        HeaderLicense.Custom(
+          s"""|/*
             | * scala-exercises - ${name.value}
             | * Copyright (C) 2015-2016 47 Degrees, LLC. <http://www.47deg.com>
```

Comment:
```suggestion
            | * Copyright (C) 2015-2019 47 Degrees, LLC. <http://www.47deg.com>
```

Very minor thing, but I think we could be updating the copyright now that we are into this.